addons/website/static/src/js/widgets/link_popover_widget.js

```diff
@@ -16,9 +16,18 @@ weWidgets.LinkPopoverWidget.include({
                 timeoutID = setTimeout(() => this.$target.popover('show'), 1500);
             });
         }
-
         return this._super(...arguments);
     },
+    async _loadAsyncLinkPreview() {
+        if(this.target.href){
+            let url = new URL(this.target.href);
+            if(url.protocol.startsWith('http') && window.location.hostname === url.hostname)
+                this.url = new URL(`/@${url.pathname}`, window.location.origin);
+            else
+                this.url = this.target.href;
```

Comment:
So here this is a text url, but the other case it is a `new URL()`, does it even work when you later do 
```
const clipboard = new _ClipboardJS(
            this.$copyLink[0],
            {text: () => this.url}
```
?